base/applications/sdbinst/sdbinst.c

```diff
@@ -0,0 +1,957 @@
+/*
+ * PROJECT:     ReactOS sdbinst
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Application compatibility database installer
+ * COPYRIGHT:   Copyright 2020 Max Korostil (mrmks04@yandex.ru)
+ */
+
+
+#include <windef.h>
+#include <winbase.h>
+#include <tchar.h>
+#include <winreg.h>
+#include <strsafe.h>
+#include <objbase.h>
+#include <apphelp.h>
+
+
```

Comment:
```suggestion
#include <tchar.h>

#include <windef.h>
#include <winbase.h>
#include <winreg.h>
#include <strsafe.h>
#include <objbase.h>
#include <apphelp.h>
```
(separating the CRT headers from the Win32 ones, and removing extra newlines.)